python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -172,8 +180,13 @@ def fetch_trial_dataframes(self) -> Dict[str, DataFrame]:
         fail_count = 0
         for path in self._get_trial_paths():
             try:
-                self.trial_dataframes[path] = pd.read_csv(
-                    os.path.join(path, EXPR_PROGRESS_FILE))
+                if self.file_type == "json":
+                    with open(os.path.join(path, EXPR_RESULT_FILE), "r") as f:
+                        json_list = [json.loads(line) for line in f]
```

Comment:
I wrote something similar recently and I had to implement it like below because some of the results files had empty lines and the json.loads would except on it.

`json_list = [json.loads(line) for line in f if line]`